package.json

```diff
@@ -160,13 +160,13 @@
         "@govflanders/vl-ui-progress-bar": "14.0.2",
         "@govflanders/vl-ui-util": "14.0.2",
         "@swc/helpers": "0.5.3",
-        "@lit-labs/react": "2.1.1",
+        "@lit-labs/react": "2.1.2",
```

Comment:
Deze experimentele package is niet meer nodig gezien vanaf Lit 3.0, nu is er een "gepromoveerde" `lit/react` beschikbaar

https://lit.dev/blog/2023-10-10-lit-3.0/